src/LinuxTracing/PerfEventOpen.h

```diff
@@ -102,6 +102,7 @@ static constexpr uint64_t SAMPLE_REGS_USER_SP_IP_ARGUMENTS =
 //  sample, this constant should be a parameters and should be made available in
 //  some setting.
 static constexpr uint16_t SAMPLE_STACK_USER_SIZE = 65000;
+static constexpr uint16_t SAMPLE_STACK_USER_SIZE_ON_CALLCHAINS = 128;
```

Comment:
Why exactly 128? Is this arbitrary? Either way, also add a comment about the reasoning.